### PR TITLE
Updated Autoscaling our Applications and Clusters module to accommodate EKS 1.20 upgrade. 

### PR DESCRIPTION
**Pull request by sathish2304**
*Issue #, if available:*  As per the request from Trevor Roberts , Updated Autoscaling our Applications and Clusters module to accommodate EKS 1.20 upgrade.

*Description of changes:* 

Updated metrics server from release v0.4.1 to v0.5.0 which is compatiable from Kubernetes 1.8+

Deploy CA 
- Deploy CA : Updated Describe autoscaling group output inline with EKS-Workshop Blueprint from 3 3 3 to 2 4 3 
- Deploy CA : Cluster Autoscaler upgraded from v1.14.7 to v1.21.0
- Deploy CA : Updated Memory Limits of Cluster Autoscaler from 300Mi to 500Mi

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: content/beginner/080_scaling/deploy_ca.files/cluster-autoscaler-autodiscover.yaml [hunk 1]
```diff
@@ -138,15 +138,15 @@ spec:
     spec:
       serviceAccountName: cluster-autoscaler
       containers:
-        - image: k8s.gcr.io/cluster-autoscaler:v1.14.7
+        - image: k8s.gcr.io/cluster-autoscaler:v1.21.0
```
Comment (buzzsurfr) [inline content/beginner/080_scaling/deploy_ca.files/cluster-autoscaler-autodiscover.yaml:141]: The PR title says 1.20 but this container image is for 1.21. Please change to 1.20.

**Review by buzzsurfr — COMMENTED** [review]
Version conflict between PR title and container image. Set to 1.20 and we'll do 1.21 once it's default in eksctl.

File: content/beginner/080_scaling/deploy_ca.files/cluster-autoscaler-autodiscover.yaml (hunk 1 above)
Comment (sathish2304) [reply content/beginner/080_scaling/deploy_ca.files/cluster-autoscaler-autodiscover.yaml]: New Commit submitted after modification based on the review comments.

**Review by buzzsurfr — APPROVED** [review]
LGTM :shipit:

**Pull request merged**